pydis_site/apps/content/resources/guides/python-guides/app_commands.md

```diff
@@ -0,0 +1,448 @@
+# DISCORD.PY RESUMATION CHANGES
+
+---
+
+Upon resumption of the most popular discord API wrapper library for python, `discord.py`, while catching on to the latest features of the discord API, there have been numerous changes with addition of features to the library. Some additions to the library are -> Buttons support, Select Menus Support, Forms (AKA Modals), Slash Commands (AKA Application Commands) and a bunch of more handy features! All the changes can be found [here](https://discordpy.readthedocs.io/en/latest/migrating.html). Original discord.py gist regarding resumation can be found [here](https://gist.github.com/Rapptz/c4324f17a80c94776832430007ad40e6).
+
+# Why this gist?
+
+---
+
+This Gist is being created as an update to Slash Commands (app commands) with explanation and examples. This Gist mainly focuses on **SLASH COMMANDS** for discord.py 2.0 (and above)!
+
+# What Are Slash Commands?
+
+---
+
+Slash Commands are the exciting new way to build and interact with bots on Discord. With Slash Commands, all you have to do is type / and you're ready to use your favourite bot. You can easily see all the commands a bot has, and validation and error handling help you get the command right the first time.
```

Comment:
I suggest we change this to
As soon as you type "/", you can easily see all the commands a bot has. It also comes with autocomplete, validation and error, which will all help you get the command right the first time.